webrender/src/util.rs

```diff
@@ -185,19 +185,19 @@ pub fn subtract_rect(rect: &Rect<f32>,
             let ox1 = ox0 + int.size.width;
             let oy1 = oy0 + int.size.height;
 
-            let r = rect_from_points_f(rx0, ry0, ox0, ry1);
+            let r = TypedRect::from_untyped(&rect_from_points_f(rx0, ry0, ox0, ry1));
```

Comment:
hmm, it's strange that you need `from_untyped` here, given that all the inputs are already typed by `U`